gabbi/tests/test_load_data_file.py

```diff
@@ -0,0 +1,76 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+"""Test loading data from files with <@.
+"""
+
+import unittest
+
+from gabbi import case
+from six.moves import mock
+
+
+@mock.patch(
```

Comment:
This mock fails on python 3.3 and 3.4. I _think_ that doing a `create=True` will fix it. See [the note in the docs](https://docs.python.org/dev/library/unittest.mock.html).